FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Basic_TLS/demo_config.h

```diff
@@ -60,7 +60,12 @@
  * must be unique; so edit as required to ensure that no two clients connecting to
  * the same broker use the same client identifier.
  *
- * #define democonfigCLIENT_IDENTIFIER    "...insert here..."
+ *!!! Please note a #defined constant is used for convenience of demonstration
+ *!!! only.  Production devices can use something unique to the device that can
+ *!!! be read by software, such as a production serial number, instead of a
+ *!!! hard coded constant.
+ *
```

Comment:
can we add a line in the comment here reading something like 
// e.g. #define democonfigCLIENT_IDENTIFIER readThingName()

Reply:
+1

Reply:
Can do in next iteration.